src/sentry/api/endpoints/organization_health.py

```diff
@@ -225,6 +225,25 @@ def get(self, request, organization):
 
 class OrganizationHealthGraphEndpoint(OrganizationHealthEndpointBase):
 
+    def get_environments(self, request, organization):
+        requested_environments = set(request.GET.getlist('environment'))
```

Comment:
Since we're not handling "no environment", I'd just suggest we at least error or something, otherwise it's going to not work when passed to Snuba.